tests/gis_tests/test_geoforms.py

```diff
@@ -203,12 +203,17 @@ def assertMapWidget(self, form_instance):
         self.assertIn('map_srid: 4326,', rendered)
         self.assertIn('gis/js/OLMapWidget.js', str(form_instance.media))
 
-    def assertTextarea(self, geom, rendered):
+    def assertTextarea(self, geom, rendered, include_map_srid=True):
         """Makes sure the wkt and a textarea are in the content"""
 
         self.assertIn('<textarea ', rendered)
         self.assertIn('required', rendered)
-        self.assertIn(escape(geom.json), rendered)
+        expected = geom.json
+        if include_map_srid:
```

Comment:
Some test failures remain. It looks like this assertion needs to transform `expected` into the new SRID similar to the change on 119. What code can do that?